lcs/Perception.py

```diff
@@ -27,3 +27,12 @@ def __len__(self) -> int:
 
     def __repr__(self):
         return ' '.join(map(str, self))
+
+    def __setitem__(self, i, o):
```

Comment:
Tutaj nie możemy tego zrobić.

1. Percepcja z definicji nie może być zmieniona
2. Ta klasa rozszerza `collections.abc.Sequence`. Proszę zerknąć na https://docs.python.org/3/library/collections.abc.html - tam jest opis poszczególnych klas.

Reply:
Dobrze, w takim razie będę musiała zrobić kilka funkcji w inny sposób, bo potrzebne było tworzenie zmienionego Perception, ale da się to zrobić inaczej.